jupyter_lidar_inspection.ipynb

```diff
@@ -0,0 +1,918 @@
+    "# See computational region\n",
+    "gs.run_command(\"g.region\",flags=\"p\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61b147de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the resolution\n",
+    "gs.run_command(\"g.region\",res=1) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffbbbd92",
+   "metadata": {},
+   "source": [
+    "We are interested to see how different filters, gridding (resolution) and methods affects our raszerization\n",
+    "Our point cloud has no CRS defined but it fits our project projection, se we overrired the projection chech with `-o`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c394f189-30fe-4bc3-b9a6-90570e1d9733",
+   "metadata": {},
+   "source": [
+    "Import our point cloud as a raster DSM, so we can check it a bit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4078e3f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "# With -e we use the extent of the input for the raster extent \n",
+    "# With -n we set the computation region to match the new raster map\n",
+    "\n",
+    "gs.run_command(\"r.in.pdal\",\n",
+    "               input=point_cloud,\n",
+    "               output='point_cloud_1m',\n",
+    "               resolution=1,\n",
+    "               flags=\"eno\")\n",
+    "\n",
+    "gs.run_command(\"r.in.pdal\",\n",
+    "               input=point_cloud,\n",
+    "               output='point_cloud_1m_last',\n",
+    "               resolution=1,\n",
+    "               return_filter='last', # Let's see what points are last\n",
+    "               flags=\"eo\")\n",
+    "\n",
+    "gs.run_command(\"r.in.pdal\",\n",
+    "               input=point_cloud,\n",
+    "               output='point_cloud_1m_ground',\n",
+    "               resolution=1,\n",
+    "               class_filter=2, # Let's see what points are classified as ground\n",
+    "               flags=\"eo\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fcfc296f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a list of the rasters we produced\n",
+    "point_cloud_check=gs.list_strings(type=['raster'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93346e2b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "series = gj.SeriesMap(height = 700)\n",
+    "series.add_rasters(point_cloud_check)\n",
+    "series.d_grid(size=250,color='orange') # With a grid will be easier to focus later on checking specific parts\n",
+    "series.show()\n",
+    "\"series.save(\"/your/path/image.gif\") # You can export that into a GIF, add it to your presentation and be cool"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d3b999c1",
+   "metadata": {},
+   "source": [
+    "Set a smaller computational region for checking how would we like the data processed"
```

Comment:
This is a bit misleading, as the next code cell actually saves the current region instead of setting a new smaller one